package.json

```diff
@@ -47,8 +47,9 @@
     "prepare-release": "babel-node scripts/release/prepare-release.js",
     "check": "babel-node scripts/run-check.js",
     "docker:build": "docker build -t leonai/leon .",
-    "docker:run": "docker run -e TZ=`cat /etc/timezone` -p 1337:1337 -it leonai/leon",
```

Comment:
Did you try to run it with a Windows OS to see if it breaks without the `TZ` environment variable?